### PR TITLE
Update lr_scheduler.py

### PR DESCRIPTION
**Pull request by 7starsea**
Sometimes, people may need to change other hyperparameter of training when the learning_rate is reduced, and it seems the good way is to update the return of the method ```ReduceLROnPlateau.step```  (it will return True if learning_rate is reduced otherwise return False).

Should we also update the return of other other learning rate schedulers' step method?

### CODE REVIEW

**Review by albanD — COMMENTED** [review]
Hi,

Thanks for this proposal but I'm not sure we will accept it for a couple reasons:
- LRScheduler is only being maintained and no core contributor works on adding significant features to it. This change, while very small in terms of code is pretty significant in terms of API and so would require a proper review.
- The step function in optimizer return the loss if a closure is passed and nothing otherwise. Having this returning a boolean value is quite unexpected here I think.
- This returned value is not very granular and it doesn't tell you which lr was actually updated.
- Can you do the same thing with a simple utility function that checks if the value of the `lr` was changed?

cc @jbschlosser

**Comment by 7starsea** [comment]
Hi,
  Thanks for the detailed response. Currently, I use the customized ReduceLROnPlateau for my own use case, and I think it is not hard to write a utility function to check if the learning rate is changed.
